curator/eslint.config.mjs

```diff
@@ -0,0 +1,23 @@
+import globals from 'globals';
+import pluginJs from '@eslint/js';
+import tseslint from 'typescript-eslint';
+import pluginReact from 'eslint-plugin-react';
```

Comment:
You should also add [eslint-plugin-react-hooks](https://www.npmjs.com/package/eslint-plugin-react-hooks). It will help to enforce rules of hooks for react.